libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -175,31 +175,6 @@ public static synchronized void reset(UserAccount account, String communityId) {
     }
 
     /**
-     * Changes the passcode to a new value and re-encrypts the
-     * stored event data with the new passcode.
-     *
-     * @param oldPass Old passcode.
-     * @param newPass New passcode.
-     */
-    public static synchronized void changePasscode(String oldPass, String newPass) {
```

Comment:
We don't need this method anymore, since passcode is not directly tied to encryption key.